package.json

```diff
@@ -40,21 +40,28 @@
     "encodeurl": "~2.0.0",
     "escape-html": "~1.0.3",
     "etag": "~1.8.1",
+    "express": "^4.19.2",
+    "express-rate-limit": "^7.2.0",
     "finalhandler": "1.2.0",
     "fresh": "0.5.2",
+    "helmet": "^7.1.0",
     "http-errors": "2.0.0",
+    "i18n": "^0.15.1",
```

Comment:
It requires at least Node@10: https://github.com/mashpie/i18n-node/blob/master/package.json#L59